app/requirements.txt

```diff
@@ -14,7 +14,6 @@ google-api-core==2.19.2
 google-auth==2.33.0
 httpx==0.27.2
 itsdangerous==2.2.0
-Jinja2==3.1.4
```

Comment:
I "think" we might need Jinja for the frontend but I could be wrong. I usually use this tool - https://pypi.org/project/pip-check-reqs/ which I find super useful to see what python dependencies are used.

Reply:
It's listed twice somehow?

Reply:
Oh weird. Thanks for checking - I guess I could have opened the file to see.